test/packages/assets_elastic_snapshot_versions_prerelease/manifest.yml

```diff
@@ -0,0 +1,34 @@
+format_version: 1.0.0
+name: assets_elastic_snapshot_versions_prerelease
+title: "Asse1s Elastic snapshot versions in Prerelease package"
+version: 0.0.1
```

Comment:
Package whose version is technical preview (0.0.1). This package is allowed to contain assets built from Elastic stack SNAPSHOT versions